js/background.js

```diff
@@ -398,17 +398,15 @@
     await storage.put('version', currentVersion);
 
     if (newVersion) {
-      if (
-        lastVersion &&
-        window.isBeforeVersion(lastVersion, 'v1.15.0-beta.5')
-      ) {
-        await window.Signal.Logs.deleteAll();
-        window.restart();
-      }
-
       window.log.info(
         `New version detected: ${currentVersion}; previous: ${lastVersion}`
       );
+
+      await window.Signal.Data.cleanupOrphanedAttachments();
+
+      await window.Signal.Logs.deleteAll();
+
+      window.restart();
```

Comment:
No need to restart the application :)

Reply:
Are you sure?

Reply:
I would assume the restart was so that the db migration would trigger. But could you check if you need a restart if you delete the logs?

Reply:
I guess not, let me check real quick.

Reply:
You were right